content/homepage.js

```diff
@@ -184,12 +184,184 @@ const tools = {
+                {
+                    "class": w48,
+                    "site": "https://www.maxar.com/",
+                    "image": "maxar-technologies.png",
+                    "alt": "maxar technologies logo",
+                },
+                {
+                    "class": w48,
+                    "site": "https://arturo.ai/",
+                    "image": "arturo.jpeg",
+                    "alt": "Arturo logo",
+                },
+                {
+                    "class": w48,
+                    "site": "https://www.usgs.gov/",
+                    "image": "usgs.png",
+                    "alt": "usgs logo",
+                },
+                {
+                    "class": w48,
+                    "site": "https://www.cosmiqworks.org/",
+                    "image": "cosmiqworks.png",
+                    "alt": "cosmiqworks logo",
+                },
+            ],
+
+            [
+                {
+                    "class": "md:w-32 m-3 p-2 md:p-4",
+                    "site": "https://aws.amazon.com/",
+                    "image": "aws.png",
+                    "alt": "aws logo",
+                },
+                {
+                    "class": w32,
+                    "site": "https://www.digitalearthafrica.org/",
+                    "image": "digital-earth-africa.png",
+                    "alt": "digital earth africa logo",
+                },
+                {
+                    "class": w32,
+                    "site": "https://www.worldbank.org/",
+                    "image": "world-bank-group.png",
+                    "alt": "world bank group logo",
+                },
+                {
+                    "class": w32,
+                    "site": "https://www.azavea.com/",
+                    "image": "azavea.png",
+                    "alt": "azavea logo",
+                },
+                {
+                    "class": w32,
+                    "site": "https://www.nasa.gov/",
+                    "image": "nasa.png",
+                    "alt": "NASA logo",
+                },
+            ],
+
+            [
+                {
+                    "class": w32,
+                    "site": "https://umbra.space/",
+                    "image": "umbra.png",
+                    "alt": "Umbra logo",
+                },
+                {
+                    "class": w32,
+                    "site": "https://openeo.org/",
+                    "image": "open-eo.png",
+                    "alt": "Open EO logo",
+                },
+                {
+                    "class": w32,
+                    "site": "", // what is this company? could find their site
+                    "image": "pixel8.svg",
+                    "alt": "Pixel 8 logo",
+                },
+                {
+                    "class": w32,
+                    "site": "https://en.wikipedia.org/wiki/China%E2%80%93Brazil_Earth_Resources_Satellite_program", // didn't find a better site
```

Comment:
Attention pls.

Reply:
Yeah, this is what I'd link to, I don't know any better site.